libs/template/templates/default-python/template/{{.project_name}}/src/{{.project_name}}/tasks/sample_etl_task.py.tmpl

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+from sklearn.datasets import fetch_california_housing
+from {{.project_name}}.common import Task
+
+
+class SampleETLTask(Task):
```

Comment:
This is based on what's in dbx, but I'm not really happy with these as-is and plan to scrap most of the code here. We should separate examples and this template.